pkg/lspext/cacheext.go

```diff
@@ -0,0 +1,23 @@
+package lspext
+
+// CacheGetParams is the input for 'cache/xget'. The response is a
+// 'CacheItem'. This cache is global to a language server. IE it is shared
+// amongst workspaces, but not amongst different language server types.
+type CacheGetParams struct {
+	Key string `json:"key"`
+}
+
+// CacheItem is the response for 'cache/xget'.
+type CacheItem struct {
+	// Ok is true if the item was successfuly retrieved from the cache.
+	Ok bool `json:"ok"`
```

Comment:
Definitely. But wouldn't omitting `value` / returning `null` for this be enough? That's what the JS Map does when the entry does not exist, it returns `undefined`